examples/angular/src/features/hacker-news-story/hacker-news-story.provider.ts

```diff
@@ -1,25 +1,24 @@
 import { CacheRepository, DefaultObjectValidator, GetInteractor, GetRepository, InMemoryDataSource, RepositoryMapper, SingleGetDataSourceRepository, VoidDataSource } from '@mobilejazz/harmony-core';
-
-import { HackerNewsStoryNetworkDataSource } from 'src/business-logic/data/data-sources/hacker-news-story.network.data-source';
-import { HackerNewsStoryEntity } from 'src/business-logic/data/entities/hacker-news-item.entity';
-import { HackerNewsStoryJSONToHackerNewsStoryEntityMapper } from '../data/mappers/hacker-news-story.mapper';
-import { HackerNewsFetchService, HackerNewsService } from '../data/service/hacker-news.service';
-import { GetHackerNewsLatestAskStoriesInteractor } from './interactors/get-hacker-news-latest-ask-stories.interactor';
-import { GetHackerNewsStoryInteractor } from './interactors/get-hacker-news-story.interactor';
-import { HackerNewsStoryEntityToHackerNewsStoryMapper, HackerNewsStoryToHackerNewsStoryEntityMapper } from './mappers/hacker-news-story.mapper';
-import { HackerNewsStory } from './models/hacker-news-story.model';
-import { CacheDecoratorFactory } from './utils';
-import { HackerNewsStoryIdsNetworkDataSource } from "../data/data-sources/hacker-news-story-ids.network.data-source";
+import { HackerNewsStoryNetworkDataSource } from 'src/features/hacker-news-story/data/data-sources/hacker-news-story.network.data-source';
+import { HackerNewsStoryEntity } from 'src/features/hacker-news-story/data/entities/hacker-news-item.entity';
+import { HackerNewsStoryJSONToHackerNewsStoryEntityMapper } from './data/mappers/hacker-news-story.mapper';
+import { HackerNewsFetchService, HackerNewsService } from './data/service/hacker-news.service';
+import { GetHackerNewsLatestAskStoriesInteractor } from './domain/interactors/get-hacker-news-latest-ask-stories.interactor';
+import { GetHackerNewsStoryInteractor } from './domain/interactors/get-hacker-news-story.interactor';
+import { HackerNewsStoryEntityToHackerNewsStoryMapper, HackerNewsStoryToHackerNewsStoryEntityMapper } from './domain/mappers/hacker-news-story.mapper';
+import { HackerNewsStory } from './domain/models/hacker-news-story.model';
+import { HackerNewsStoryIdsNetworkDataSource } from "./data/data-sources/hacker-news-story-ids.network.data-source";
+import {CacheDecoratorFactory} from "@bugfender/sdk/lib/utils";
```

Comment:
I don't think you wanted to import this. The import you want is:

```
import {createCacheDecorator} from '@mobilejazz/harmony-core';
```

And use it like:

```
const Cached = createCacheDecorator();
```

Reply:
Sure! I'll change that, it was an automatic import. We should remove the one inside bugfender if possible

Reply:
Bugfender it's not exporting it from the root… I'm not sure if that can be fixed.